diff -Naur a/node_modules/bip38/index.js b/node_modules/bip38/index.js
--- a/node_modules/bip38/index.js	2015-11-24 18:02:18.000000000 +0100
+++ b/node_modules/bip38/index.js	2015-11-24 18:04:58.000000000 +0100
@@ -2,7 +2,7 @@
 var assert = require('assert')
 var createHash = require('create-hash')
 var cs = require('coinstring')
-var scrypt = require('scryptsy')
+var scrypt = function() { return Bitcoin.scrypt.apply(this, arguments); }
 var xor = require('buffer-xor')
 
 var ecurve = require('ecurve')
